modules/administration-guide/pages/installing-che-locally.adoc

```diff
@@ -24,6 +24,6 @@ To run and manage {prod-short}:
 
 Choose one of the following procedures to start the {prod-short} Server using the `{prod-cli}` tool.
 
-* xref:installing-che-on-codeready-containers.adoc[]
+* xref:installing-che-on-devspaces.adoc[]
```

Comment:
The new page be "openshift-local" ... You can't install Che on Dev Spaces since Dev Spaces is BUILT from Che.